app/models/project.rb

```diff
@@ -138,11 +138,23 @@ def send_notifications
   end
 
   def subjects_count
-    @subject_count ||= live_subject_sets.sum :set_member_subjects_count
+    @subject_count ||= if live_subject_sets.loaded?
+      live_subject_sets.inject(0) do |sum,set|
```

Comment:
Alternatively: `live_subject_sets.to_a.sum(&:set_member_subjects_count)`. The `to_a` is a little bit hidden in there though.